Scrum 107 add contact us page

Alice626Moli opened the pull request:
I wrote a basic contactus page with test and storybook files, all works.
This work is separated from SCRUM-81 Create a DatePicker, I will fix conflicts if we made sure about the latest adjustment and changes to SCRUM -81, the preview.tsx and other changes.

Alice626Moli commented:
https://github.com/user-attachments/assets/84078a0a-22f1-47bb-a914-571261eb76a5
I changed the contactus page with import inputbox component and used its validation rules.
I added a 'message' type in InputBox component, and changed its storybook, utils and test file as well. 
All test and storybooks can run.
A simple video demo has now been attacehd.

DAHUO-Melbourne commented on __tests__/ContactUs.test.tsx:
contact us是一个page，他不应该有unit test，他对应的测试是e2e test

DAHUO-Melbourne commented on src/components/ContactUs.tsx:
contact us应该是一个page，而不是一个component，你应该把他放在app folder里然后在navbar里新增一个item这样用户点击就能跳转到这一个页面里

Alice626Moli commented:
Installed the @playwright/test dependency and added related scripts, and set up the corresponding playwright.config.ts file.

Added a Contact Us button and Link component in the NavBar to enable navigation via the button.

Created an e2e folder and wrote an E2E test file for Contact Us.

Considering global style adjustments and consistency, no further style changes were made to the Contact Us file as a secondary page.

Configured the related translation files.

The contact us page now can be translated from en to zh once used the switch button.
